### PR TITLE
Some random improvements from recent sweeper work

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

File: .python-version [hunk 1]
```diff
@@ -1 +1 @@
-3.11
```
Comment (steveoh) [inline .python-version:1]: **issue**: pro is going to be 3.11. i think that is why this is 11 instead of 12.

Comment (stdavis) [reply .python-version]: OK, Pro-bro. ;) I removed this commit.

**Review by steveoh — APPROVED** [review]

**Comment by stdavis** [comment]
I'm going to leave this for @jacobdadams's review. No rush.

**Review by jacobdadams — APPROVED** [review]
This all looks good to me.

**Pull request merged**